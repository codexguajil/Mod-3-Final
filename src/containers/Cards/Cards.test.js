import Card from './Cards'
import React from 'react'
import { shallow } from 'enzyme'

describe('Card', () => {
  let wrapper;

  it('should match the snapshot', () => {
    wrapper = shallow(
      <Card />
    )

    expect(wrapper).toMatchSnapshot()
  })
})