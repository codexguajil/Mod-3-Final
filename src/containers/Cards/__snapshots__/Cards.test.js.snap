// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Card />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2 />,
        <h3>
          Founded: 
        </h3>,
        <p>
          Seats: 
        </p>,
        <p>
          Titles: 
        </p>,
        <p>
          Coat of Arms: 
        </p>,
        <p>
          Ancestral Weapons: 
        </p>,
        <p>
          Words: 
        </p>,
      ],
      "className": "card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Founded: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Founded: ",
          undefined,
        ],
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Seats: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Seats: ",
          undefined,
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Titles: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Titles: ",
          undefined,
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Coat of Arms: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Coat of Arms: ",
          undefined,
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Ancestral Weapons: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Ancestral Weapons: ",
          undefined,
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Words: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Words: ",
          undefined,
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2 />,
          <h3>
            Founded: 
          </h3>,
          <p>
            Seats: 
          </p>,
          <p>
            Titles: 
          </p>,
          <p>
            Coat of Arms: 
          </p>,
          <p>
            Ancestral Weapons: 
          </p>,
          <p>
            Words: 
          </p>,
        ],
        "className": "card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Founded: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Founded: ",
            undefined,
          ],
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Seats: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Seats: ",
            undefined,
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Titles: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Titles: ",
            undefined,
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Coat of Arms: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Coat of Arms: ",
            undefined,
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Ancestral Weapons: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Ancestral Weapons: ",
            undefined,
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Words: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Words: ",
            undefined,
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
